docs/source/_toctree.yml

```diff
@@ -1,14 +1,10 @@
 - sections: 
   - local: index
     title: 🤗 Transformers
-  - local: quicktour
-    title: Quick tour
```

Comment:
Hmm interesting, I was thinking it'd be better to show how to install so the user is all set up before doing the quicktour. Not a big deal either way though since we can just add `pip install transformers` to the quicktour. 👍